### PR TITLE
fix: update production scripts and deleted unused dependencies in roo…

### PR DESCRIPTION
**Pull request by cmcWebCode40**
- [x] update production scripts in package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "dependencies": {
-    "@verida/account-node": "^1.1.9"
-  }
+	"dependencies": {
+		"@verida/account-node": "^1.1.9"
+	}
```
Comment (dmty) [inline package.json:4]: New prettier config? :)

**Review by dmty — APPROVED** [review]
What will happen with all that GH Actions warnings?

File: package.json (hunk 1 above)
Comment (cmcWebCode40) [reply package.json:4]: No, will fix this

**Comment by cmcWebCode40** [comment]
> What will happen with all that GH Actions warnings?

This is primarily a DEPRECATION WARNING from the sass and vuetify lib. 

I can look deeper into it to see how to fix them later. 
that will be a different ticket.

**Pull request merged**